LiteDB/Client/Database/LiteDatabase.cs

```diff
@@ -56,14 +56,17 @@ public LiteDatabase(ConnectionString connectionString, BsonMapper mapper = null)
         }
 
         /// <summary>
-        /// Starts LiteDB database using a generic Stream implementation (mostly MemoryStrem).
-        /// Use another MemoryStrem as LOG file.
+        /// Starts LiteDB database using a generic Stream implementation (mostly MemoryStream).
         /// </summary>
-        public LiteDatabase(Stream stream, BsonMapper mapper = null)
+        /// <param name="stream">DataStream reference </param>
+        /// <param name="mapper">BsonMapper mapper reference</param>
+        /// <param name="logStream">LogStream reference </param>
+        public LiteDatabase(Stream stream, BsonMapper mapper = null, Stream logStream = null)
         {
             var settings = new EngineSettings
             {
-                DataStream = stream ?? throw new ArgumentNullException(nameof(stream))
+                DataStream = stream ?? throw new ArgumentNullException(nameof(stream)),
+                LogStream = logStream
```

Comment:
I think this breaks when `logStream` is null. Maybe you should make it `LogStream = logSteam ?? new MemoryStream()`

Reply:
Hi @lbnascimento thank you for feedback and code review.

Just double checked it and I think additional checking for log stream is not necessary at this point.  Please have a look on following code path call :

LiteEngine cotr , line 70 :
```csharp
                _disk = new DiskService(settings, MEMORY_SEGMENT_SIZES);
```
DiskService ctro,  line 37 :
```csharp
            _logFactory = settings.CreateLogFactory();
```
and CreateLogFactory method line 108 : 
```csharp
            return new StreamFactory(new MemoryStream(), this.Password);
```

In CreateLogFactory factory method if logStream is null StreamFactory will be instances with new MemoryStream()

Let me know what you think about it and if I didn't miss anything. Still learning LiteDB :)

Regards
Piotr